### PR TITLE
Update free-courses-hi.md

### PR DESCRIPTION
**Pull request by ShashankSagarJha**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](/CONTRIBUTING.md)
- [ ] Search for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists  in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

File: free-courses-hi.md [hunk 1]
```diff
@@ -46,3 +46,9 @@
 ### Python
 
 * [Python Tutorial For Beginners | Hindi (With Notes)](https://www.youtube.com/playlist?list=PLu0W_9lII9agICnT8t4iYVSZ3eykIAOME) - Harry
+
+### Dynamic Programming
+
+* [Dynamic Programming in Hindi| Detailed Analysis from recursive equation to Bottom Up]  (https://www.youtube.com/playlist?list=PLeF0b8iqbx4mogykbd82-HY9Y1-JS9MDr) Shashank Jha
```
Comment (SethiPandi) [inline free-courses-hi.md:52]: Hey, and welcome to free programming books! :wave:

* Please use the title from the source, don't create your own.
* Markdown links should not have a space between the ] and (
* There should be a hyphen with spaces around before the author/vendor.
* Not sure if dynamic programming needs to be its own section, but could you please add a heading for it while it's here?

```suggestion
* [Dynamic Programming in Hindi](https://www.youtube.com/playlist?list=PLeF0b8iqbx4mogykbd82-HY9Y1-JS9MDr) - Shashank Jha
```

Comment (SethiPandi) [reply free-courses-hi.md]: I didn't notice until now, could you please also ensure new sections have two empty lines above them.

**Comment by SethiPandi** [comment]
> Not sure if dynamic programming needs to be its own section, but could you please add a heading for it while it's here?

Sorry, I meant could you please add it to the table of contents at the top.*

**Comment by ShashankSagarJha** [comment]
Done.
Please let me know.
I will be creating different playlist for different algorithmic techniques

Thanks,
Shashank Jha

**Comment by eshellman** [comment]
looks like there are no changes left

**Pull request closed without merging**